### PR TITLE
contrib/aws/aws-sdk-go: expose handler names as const

### PR DESCRIPTION
**Pull request by mstumpfx**
I am directly modifying handlers in the request stack, and referencing them `ByName` will be safer if these are exposed. 

Reference: https://github.com/DataDog/dd-trace-go/issues/1168#issuecomment-1284210112

### CODE REVIEW

**Comment by knusbaum** [comment]
I don't see any reason *not* to expose these.

**Review by knusbaum — APPROVED** [review]

File: contrib/aws/aws-sdk-go/aws/aws.go [hunk 1]
```diff
@@ -20,11 +20,13 @@ import (
...
-	tagAWSRegion     = "aws.region"
-	tagAWSRetryCount = "aws.retry_count"
-	tagAWSRequestID  = "aws.request_id"
+	tagAWSAgent         = "aws.agent"
+	tagAWSOperation     = "aws.operation"
+	tagAWSRegion        = "aws.region"
+	tagAWSRetryCount    = "aws.retry_count"
+	tagAWSRequestID     = "aws.request_id"
+	SendHandlerName     = "gopkg.in/DataDog/dd-trace-go.v1/contrib/aws/aws-sdk-go/aws/handlers.Send"
+	CompleteHandlerName = "gopkg.in/DataDog/dd-trace-go.v1/contrib/aws/aws-sdk-go/aws/handlers.Complete"
```
Comment (knusbaum) [inline contrib/aws/aws-sdk-go/aws/aws.go:29]: These need comments documenting what they are.

**Review by knusbaum — CHANGES_REQUESTED** [review]
Ahh, forgot this nit.

File: contrib/aws/aws-sdk-go/aws/aws.go (hunk 1 above)
Comment (mstumpfx) [reply contrib/aws/aws-sdk-go/aws/aws.go]: Sure, thanks. Done, happy to revise the language if desired.

**Review by knusbaum — APPROVED** [review]
LGTM.

File: contrib/aws/aws-sdk-go/aws/aws.go (hunk 1 above)
Comment (knusbaum) [reply contrib/aws/aws-sdk-go/aws/aws.go]: Looks good, thanks.

**Comment by knusbaum** [comment]
Tests actually passed, they just failed to upload some results. This is fixed in another PR, but we don't have to wait for it.

**Comment by mstumpfx** [comment]
Thanks for the quick review!

**Pull request merged**